packages/storefront/src/components/ComponentMetaOverview.vue

```diff
@@ -45,7 +45,94 @@
     }
 
     get body(): string {
```

Comment:
The whole function is way too complex. Consider creating helper functions for the constants which require complex conditions.
Consider these best practices for your refactor: 
- Try to avoid ternaries as they are harder to read/understand. 
- Try to avoid such cases `Object.entries(propsMeta ?? {}).map` since the map function will be called at least once when `propsMeta` is `undefined` or `null` which is useless.
- `if/elseif/elseif/else` could be avoided by using returns in if-conditions inside of a function
- Move typings out of the function to the top of the file eg. `<
        T extends { isDeprecated?: boolean; isBreakpointCustomizable?: boolean; isExperimental?: boolean },
      >(`

Remember that other colleagues have to understand it so it's your challenge to support them by writing clean and structured code which is easily understandable. You already did a great job reducing the complexity after rewriting but there is more potential.